website/docs/index.html.markdown

```diff
@@ -70,7 +75,7 @@ provider "helm" {
 }
 ```
 
-### Statically defined credentials
+### Credentials config
 
 You can also **statically** define all the credentials:
```

Comment:
I wonder if the word **statically** could be misinterpreted here, when you're able to use variables.  Should it be **explicitly** or **discretely** instead?

Reply:
Yeah I was just thinking that – I think explicit makes more sense as being able to source these attributes from variables or outputs from other resources is not static at all